src/libcore/ops/try.rs

```diff
@@ -15,6 +15,7 @@
 /// extracting those success or failure values from an existing instance and
 /// creating a new instance from a success or failure value.
 #[unstable(feature = "try_trait", issue = "42327")]
+#[rustc_on_unimplemented = "the `?` operator can only be used in function that returns types that implement `std::ops::Try`, like `Result`"]
```

Comment:
"in function that returns" should be "in a function that returns" or "in functions that return"

Also since this is aimed at beginners, perhaps put the more common answer in front? "in functions that return `Result` (or another type implementing `std::ops::Try`)"